app/src/main/java/com/example/movopfy/features/home/presentation/viewmodel/HomeViewModel.kt

```diff
@@ -2,29 +2,87 @@ package com.example.movopfy.features.home.presentation.viewmodel
 
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
+import com.example.movopfy.common.mappers.mapAnilibriaUrlToImageUrl
 import com.example.movopfy.features.home.domain.models.WaitingListToday
-import com.example.movopfy.features.home.domain.usecase.GetWaitingListTodayUseCase
+import com.example.movopfy.features.home.domain.repository.AnilibriaRepository
+import com.example.movopfy.features.home.domain.repository.KinopoiskRepository
+import com.example.movopfy.network.kinopoisk.models.KinopoiskDocs
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
 import kotlinx.coroutines.flow.asStateFlow
 import kotlinx.coroutines.launch
 
-class HomeViewModel(private val getWaitingListTodayUseCase: GetWaitingListTodayUseCase) :
+class HomeViewModel(
+    private val anilibriaRepository: AnilibriaRepository,
+    private val kinopoiskRepository: KinopoiskRepository
+) :
     ViewModel() {
 
     private val _uiState = MutableStateFlow<HomeUiState>(HomeUiState.Loading)
     val uiState: StateFlow<HomeUiState> = _uiState.asStateFlow()
 
+    private var anime = emptyList<WaitingListToday>()
+    private var movieList = mutableListOf<Pair<String, List<KinopoiskDocs>>>()
+
     init {
         viewModelScope.launch {
-            val schedules = getWaitingListTodayUseCase.execute(day = 6)
+            anime =
+                mapAnilibriaUrlToImageUrl(anilibriaSchedule = anilibriaRepository.getSchedule()[6])
```

Comment:
`6` -  is a magic number. Please move to a constant. https://en.wikipedia.org/wiki/Magic_number_(programming)#:~:text=In%20computer%20programming%2C%20a%20magic,see%20List%20of%20file%20signatures

Reply:
sorry, it is not a magic number, this number I will get from local date. This should be the current day

Reply:
I forgot change it, this is my mistake

Reply:
I did it